link_shortener/templates/delete_link.html

```diff
@@ -0,0 +1,13 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <br>
+    <h1>Are you sure you know what you're doing?</h1>
+        <h2>Like really?</h2>
+    <h3>If so, then click on to kill it, else..ruuuuun</h3>
```

Comment:
No tyhle custom zprávy projdou přeze mě, ale obávám se že tam stejně pak budeš dávat něco generickýho až to uvidí marketing 😄

Reply:
Hele tie HTMLka su len tak, pre Duca a otestovanie. :)